servers/rest-services/src/main/java/org/projectnessie/services/restjakarta/NessieExceptionMapper.java

```diff
@@ -82,13 +83,18 @@ public Response toResponse(Exception exception) {
     } else if (exception instanceof NotSupportedException) {
       errorCode = ErrorCode.UNSUPPORTED_MEDIA_TYPE;
       message = exception.getMessage();
+    } else if (exception instanceof WebApplicationException
```

Comment:
Sadly a "complicated" condition here, so it works with both Quarkus (RESTeasy) and Jersey.
Didn't want to have different code paths for javax + jakarta.